svm/src/account_loader.rs

```diff
@@ -270,31 +222,16 @@ fn load_transaction_accounts<CB: TransactionProcessingCallback>(
                             // Currently, rent collection sets rent_epoch to u64::MAX, but initializing the account
                             // with this field already set would allow us to skip rent collection for these accounts.
                             default_account.set_rent_epoch(RENT_EXEMPT_RENT_EPOCH);
-                            (default_account.data().len(), default_account, 0)
+                            unique_loaded_accounts.insert(*key, default_account.clone());
+                            (default_account.data().len(), default_account)
                         })
+                    } 
+                    else {
+                        let account = unique_loaded_accounts.get(key).unwrap().clone();
```

Comment:
don't `unwrap` if not necessary, it makes it really easy to break the code.

above, instead of `if !unique_loaded_accounts_contains_key` we should have
`match unique_loaded_accounts.get(key)` or something to that effect.